packages/vx-demo/src/components/charts/SimpleBar.tsx

```diff
@@ -1,19 +1,22 @@
-import React from 'react';
+import React, { FC } from 'react';
 import { Bar } from '@vx/shape';
 import { Group } from '@vx/group';
 import { PatternLines } from '@vx/pattern';
 import { letterFrequency } from '@vx/mock-data';
```

Comment:
just noting that to fix the build, you'll probably need to update this to 
`import letterFrequency, { LetterFrequency } from '@vx/mock-data/lib/mocks/letterFrequency';`

which will allow you to use the `LetterFrequency` type in the accessor functions like `const x = (d: LetterFrequency) => d.letter;`

Reply:
Actually this wasn't the reason as to why the build was failing but I was able to find out why and fixed it.